middleware/src/rpcserver/rpcserver_test.go

```diff
@@ -97,7 +97,7 @@ func NewTestingRPCServer() TestingRPCServer {
 	testingRPCServer.middlewareMock.On("RebootBase").Return(rpcmessages.ErrorResponse{Success: true})
 	testingRPCServer.middlewareMock.On("EnableRootLogin", rpcmessages.ToggleSettingArgs{ToggleSetting: true}).Return(rpcmessages.ErrorResponse{Success: true})
```

Comment:
Seems like you forgot to add the `EnablePasswordLogin` to the mock setup. This should make the CI run...

```suggestion
	testingRPCServer.middlewareMock.On("EnableRootLogin", rpcmessages.ToggleSettingArgs{ToggleSetting: true}).Return(rpcmessages.ErrorResponse{Success: true})
	testingRPCServer.middlewareMock.On("EnablePasswordLogin", rpcmessages.ToggleSettingArgs{ToggleSetting: true}).Return(rpcmessages.ErrorResponse{Success: true})
```

Reply:
Thanks, that fixed it, the Middleware builds now successfully and passes all tests. 

I updated the RPC addition checklist: https://github.com/shiftdevices/bitbox-base-internal/issues/269 with this info as well.